packages/angular-ui-components/src/ToasterController.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
```

Comment:
Probably create a separated directory, called `services` and put this file on it. Suggest to rename the file to `ToasterService.ts` or `ToasterController.service.ts`.

Create an index.ts file and add the service to him. After that in public-api export this index instead of this service.